### DISCUSSION TITLE
通知表示のタイムアウト

### DISCUSSION THREAD

**Discussion by simon-ritchie** [increments/qiita-discussions#291] [要望・アイデア・不具合]
3日前くらい？から通知の表示が1回目がタイムアウトする感じになってきております！  
タイムアウトした後にもう一度クリックすると表示できております！（この辺の負荷が高まっているかもしれません）

![image](https://user-images.githubusercontent.com/25178045/188127874-8a3866f9-2d85-4687-a0d5-289939c3d952.png)

Chromeのコンソールでは以下のような感じで表示されているようです！

```
Failed to load resource: the server responded with a status of 502 ()
```

**Comment by tanaka-asami415** [comment]
@simon-ritchie 
ご報告ありがとうございます🙏
こちら調査いたします🙇

**Comment by kyntk** [comment]
@simon-ritchie 

こちらで調査をしてみたのですが、全体として負荷が高まっているという傾向は見られませんでした。

追加の調査をしたいので、恐れ入りますが発生する環境(デバイス・OS・ブラウザのバージョン)を教えていただけますでしょうか。
その際にスマートフォンなどの他デバイスでも発生するかも確認いただけますと幸いです。

また、通知を表示しようとしてタイムアウトした正確な時刻も教えていただけますか。

**Comment by simon-ritchie** [comment]
@tanaka-asami415 
@kyntk 

調査対応ありがとうございます！

> こちらで調査をしてみたのですが、全体として負荷が高まっているという傾向は見られませんでした。

おお、では私とかの一部のユーザーだけ発生するパターンかもしれませんね・・・！

> 追加の調査をしたいので、恐れ入りますが発生する環境(デバイス・OS・ブラウザのバージョン)を教えていただけますでしょうか。
> その際にスマートフォンなどの他デバイスでも発生するかも確認いただけますと幸いです。

自宅のデスクトップ・会社のデスクトップ・スマホ問わず発生しております！  

[自宅と会社のデスクトップ]

- Windows10
- Chrome最新版もしくは1つ前のバージョン（会社の方は最新版・自宅の方は今Chromeのヘルプを見たら1つ新しいバージョンが来ておりました！バージョンは105.0.5195.53などになります）

[スマホ]

- iPhone 12 mini
- iOS15.6.1
- スマホの方もブラウザはChromeでして、今バージョン確認したら103.0.5060.63で少し古かったようです！

> また、通知を表示しようとしてタイムアウトした正確な時刻も教えていただけますか。

しばらく時間を置いてから通知を表示しようとする（寝て起きた後や仕事後にQiitaを開いた時など）と毎回タイムアウトする・・・という感じで、前回までの発生した各時間はメモしていなかったのですが、直近で再度試して再現いたしました！本日の19:37:00頃に1度発生しております（デスクトップPCの方で発生しております）。
その後はしばらく発生しなくなる・・・という状態が続いております！

---

特に全体的な負荷などになっていない・・・という場合には私の方も二回通知をクリックすればOKくらいで影響は軽微なので優先度低めで設定いただいても大丈夫かもしれません！

**Reply by kyntk** [reply]
@simon-ritchie ご回答いただきありがとうございます。
お手数をおかけして大変申し訳ないのですが、もし可能でしたら、次回通知を確認する際にChromeの開発者ツールのNetworkタブを開いて、エラーの起きているリクエストのレスポンス等の情報を、以下のサポートフォームから送っていただけないでしょうか。

https://support.qiita.com/hc/ja/requests/new

**Reply by simon-ritchie** [reply]
@kyntk 

スクショして送信させていただきました！  
お手数ですがご確認いただけますと幸いです🙇‍♂️

**Reply by tanaka-asami415** [reply]
@simon-ritchie 
ご対応いただきありがとうございます🙏
サポートフォームより返信をお送りしておりますので、お手すきの際にご確認いただけますと幸いです🙇

**Reply by simon-ritchie** [reply]
@kyntk 
@tanaka-asami415 

ご確認＆ご返信ありがとうございます！  
返信メールの内容に合わせて追加情報（ファイル）をフォームから別途送信させていただきました！

お手すきの際にご確認いただけますと幸いです🙇‍♂️

**Reply by tanaka-asami415** [reply]
@simon-ritchie 
何度もご対応いただきありがとうございました🙏
こちら原因が判明したため対応いたします！
修正完了まで、ご不便をおかけしますがお待ちいただけますと幸いです🙇

**Reply by simon-ritchie** [reply]
@tanaka-asami415 

おお、それは良かったです！
調査＆ご対応ありがとうございます・・・！🙇‍♂️

**Reply by kyntk** [reply]
@simon-ritchie 
本日昼頃に修正をしてみましたが、時間をおいてまた発生するようでしたら教えていただけますでしょうか。

**Reply by simon-ritchie** [reply]
通常時と同じ感じで瞬時に表示できるようになりました・・・！🎉🎉
ご対応ありがとうございます！🙇‍♂️
